### ISSUE TITLE
More training detatils

### ISSUE THREAD

**Issue by Youngkl0726** [chenyilun95/tf-cpn#36]
How many epochs did you use to train the models? Did you train the different models using different epochs？ I read your paper. You said, the learning rate is decreased by a factor of 2 every 10 epcohs. Is this learning strategy used for every model? Or is there some difference between training different models?

**Comment by biras11** [comment]
Hi @Youngkl0726 ! Did you manage to retrain their network and achieve similar performances? How did you train it?
